report/sections/Chapter 02 - Preliminaries/preliminaries.tex

```diff
@@ -74,15 +72,15 @@ \section*{Prisma}
 \section*{TypeScript}
 As the server-side use of \javascript{} keeps growing in popularity its inability to provide developers with access to strong type checking and compile time error checking continues to prevent it from being the go-to language.
 As a way to address these problems, \typescript{} was designed to try and bridge this gap without requiring developers to learn a new language.
-Instead, \typescript{} was developed as a syntactic superset to \javascript{} allowing developers to keep building into their already existing knowledge but with the addition of additional features. 
+Instead, \typescript{} was developed as a syntactic superset to \javascript{} allowing developers to keep building into their already existing knowledge but with additional features. 
 This also allows for all \javascript{} projects to be converted to \typescript{} projects allowing for easier movement between the two languages\cite{TypeScript}. 
 \typescript{} accomplishes this while still keeping the advantages of an interpreted language by compiling all the written instructions to its \javascript{} equivalents during the compilation process. 
 This is possible because \typescript{} itself is compiled into \javascript{}, which is an interpreted language. 
 
 \section*{Next.js}
-Next.js (Next) is an open-source React framework created by Vercel that enables react web applications to use server-side rendering as well as generate static websites. The Next framework was created in order to counter the potential drawbacks of React relying on client-side rendering such as potential security issues. By enabling websites to be fully or partially server-side rendered instead Next provides a solution to this issues\cite {Nextjs_Docks}. 
+Next.js (Next) is an open-source React framework created by Vercel that enables react web applications to use server-side rendering as well as generate static websites. The Next framework was created in order to counter the potential security issues of React, which relies on client-side rendering. Next provides a solution to this by enabling websites to be fully or partially server-side rendered\cite {Nextjs_Docks}. 
 
-Next as a framework also tries to fix another issue of React, namely that creating a website from the ground up still requires a lot of effort spending time configuring tools as well as rebuilding solutions. 
+Next as a framework also tries to fix another issue of React, namely that creating a website from the ground up still requires a lot of effort, such as spending time on configuring tools as well as rebuilding solutions. 
```

Comment:
```suggestion
Next  tries to fix another issue of React, namely that creating a website from the ground up still requires a lot of effort, such as spending time on configuring tools as well as rebuilding solutions. 
```

Reply:
We should probably refer to it as Next.js everywhere, as that's how the docs also refer to it.

BTW, double space between Next and tries.